docs/embroider-css-modules/app/templates/dashboard.hbs

```diff
@@ -1,6 +1,6 @@
 {{page-title "Dashboard"}}
 
-<Ui::Page @title="Dashboard">
+<Ui::PageNew @title="Dashboard">
```

Comment:
I ran into an oddity when extracting components to `sample-v2-addon`. Namely, I couldn't use the same component name as before.

For the `<Ui::Page>` component, running the `start` script would result in the following error:

<details>

<summary>Error message</summary>

```sh
[docs-app-for-embroider-css-modules] Build Error (PackagerRunner) in templates/dashboard.hbs
[docs-app-for-embroider-css-modules] 
[docs-app-for-embroider-css-modules] Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
```

</details>

<details>

<summary>Stack trace and error report</summary>

```sh
ERROR Summary:

  - broccoliBuilderErrorStack: ModuleNotFoundError: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/Compilation.js:2016:28
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:817:13
    at eval (eval at create (/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:448:22
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:118:11
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:689:25
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:893:8
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:1013:5
  - code: [undefined]
  - codeFrame: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
  - errorMessage: templates/dashboard.hbs: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
        at PackagerRunner (@embroider/webpack)
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: templates/dashboard.hbs
    - line: [null]
    - treeDir: [undefined]
  - message: templates/dashboard.hbs: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
        at PackagerRunner (@embroider/webpack)
  - name: Error
  - nodeAnnotation: @embroider/webpack
  - nodeName: PackagerRunner
  - originalErrorMessage: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
  - stack: ModuleNotFoundError: Module not found: Error: Can't resolve '../components/ui/page.ts' in '$TMPDIR/embroider/2fa04b/docs/embroider-css-modules/templates/dashboard.hbs'
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/Compilation.js:2016:28
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:817:13
    at eval (eval at create (/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:10:1)
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:275:22
    at eval (eval at create (/node_modules/.pnpm/tapable@2.2.1/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:9:1)
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:448:22
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:118:11
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:689:25
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:893:8
    at /node_modules/.pnpm/webpack@5.81.0/node_modules/webpack/lib/NormalModuleFactory.js:1013:5
```

</details>